index.js

```diff
@@ -81,11 +80,15 @@ function Server () {
     app.use(speechPassport.session());
 
     // configure handlebars & register it with express app
-    const hbs = expressHandlebars.create({
+    const viewsPath = Path.resolve(process.cwd(), 'node_modules/spee.ch/server/views');
+    app.engine('handlebars', expressHandlebars({
+      async: false,
```

Comment:
what will it take to allow async?

Reply:
hmm, I'm actually not sure why it is set to false other than it was probably recommended in the `express-handlebars` implementation examples.  I'll look into that.